README.md

```diff
@@ -15,15 +15,19 @@ Features:
    * [Data mapping](#data-mapping)
    * [Configuration](#configuration)
    * [Filtering](#filtering)
+   * [ExtraLabels](#adding-extra-labels)
```

Comment:
nit: i think, having "ExtraLabels" link on level-1 of this link tree if preferable - no need to repeat for each mode.

Reply:
updated.